client/package.json

```diff
@@ -37,6 +37,7 @@
     "get-available-languages": "npm run ts -- cli/get-available-languages.ts"
   },
   "dependencies": {
+    "@angular-material-components/datetime-picker": "^16.0.1",
```

Comment:
It seems like this library is not maintained and currently not compatible with angular 17.

Reply:
Seems like I'll have to do a custom implementation with the `ngx-material-timepicker`